src/storybook/components/colors-description/colors-description.jsx

```diff
@@ -25,3 +25,5 @@ export const ColorsDescription = ({ colorNames }) => {
   );
   return <Frame>{descriptions}</Frame>;
 };
+
+export default ColorsDescription;
```

Comment:
not sure we want to to export this one

Reply:
Maybe we will want to export a component which allow to document colors but this one depend on our colorsHashMap which will not benefit other storybook which want to document their custom colors.

Reply:
Removed from `published-storybook-components`